tests/profile/admin/test_profile_highlights_admin.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from django.contrib.admin.sites import AdminSite
+from django.test import SimpleTestCase, TestCase
+
+from tests.profile.factories import ProfileHighlightFactory, ProfileFactory
+from tests.datasets.factories import DatasetFactory, IndicatorFactory
+from wazimap_ng.profile.models import ProfileHighlight
+from wazimap_ng.profile.admin.admins import ProfileHighlightAdmin
+
+class MockRequest:
```

Comment:
Not saying you need it now, but are you aware of RequestFactory? https://docs.djangoproject.com/en/3.2/topics/testing/advanced/#the-request-factory